tests/Commands/ResultsUpdateAllCommand.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace RecipeScraperTests\Commands;
+
+use InvalidArgumentException;
+use RecipeScraper\Factory;
+use RecipeScraperTests\UsesTestData;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\VarExporter\VarExporter;
+
+class ResultsUpdateAllCommand extends Command
+{
+    use UsesTestData;
+
+    protected function configure()
+    {
+        $this->setName('results:update-all')
+            ->setDescription('Update all result files with content from their corresponding HTML file')
+            ->addArgument(
+                'fields',
+                InputArgument::REQUIRED,
+                'A comma separated list of result fields to update'
+            )->addArgument(
+                'host',
+                InputArgument::REQUIRED,
+                'Host to update results for, use all for all hosts'
```

Comment:
Can we extract a dedicated `results:update-host` command and drop the host argument entirely? This would be more inline with what we are currently doing for the `html:get` commands.